solr/modules/hadoop-auth/README.md

```diff
@@ -0,0 +1,29 @@
+Apache Solr Hadoop Authentication Module
+===============================
+
+Introduction
+------------
+This module implements the support for Hadoop Authentication in Apache Solr. 
+
+Building
+--------
+The Hadoop Authentication module uses the same Gradle build as the core Solr components. 
+
+To build the module, you can use
+
+```
+./gradlew :solr:modules:hadoop-auth:assemble
+```
+
+The resulting module will be placed to the libs directory, for example:
+`solr/modules/hdfs/build/libs/solr-hadoop-auth-9.0.0-SNAPSHOT.jar`
+
+To execute the module tests:
+
+```
+./gradlew :solr:modules:hadoop-auth:test
+```
+
+Usage
+-----
+Please refer to the 'Running Solr on HDFS' section of the reference guide: https://solr.apache.org/guide/running-solr-on-hdfs.html
```

Comment:
Figure out the final 9.0 ref-guide URL to put here.
I think it will be https://solr.apache.org/guide/solr/latest/deployment-guide/solr-on-hdfs.html
See nightly 10.0 guide: https://nightlies.apache.org/solr/draft-guides/solr-reference-guide-antora/solr/10_0/deployment-guide/solr-on-hdfs.html

Reply:
Addressed in 0ab392294094da79cea43dd77895683dee7b5b08